package-lock.json

```diff
@@ -0,0 +1,313 @@
+{
+  "requires": true,
```

Comment:
Can you remove locked package file ? I think this is not required.

Reply:
Also we can start clearing project. Do we need **node_modules** at root directory ?